tools/rna_tools/trna_mod/addCCA.xml

```diff
@@ -0,0 +1,22 @@
+<tool id="addCCA" name="add CCA tails to tRNA chromosomes" version="0.1.0">
```

Comment:
As a general comment to this tool ... you can archive the same with a small workflow ...

fasta-to-tabular -> add column (CCA) -> merged columns -> tabular to fasta